<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplatesDialog</class>
 <widget class="QDialog" name="TemplatesDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Parts / Templates</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>241</width>
     <height>591</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Double click on a part to add it to the circuit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>550</y>
     <width>321</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Updates the templates with the selected parts from the current circuit.</string>
   </property>
   <property name="text">
    <string>Add from current circuit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>610</y>
     <width>321</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Deletes the selected template.  There is no undo option.</string>
   </property>
   <property name="text">
    <string>Delete selected template</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>40</y>
     <width>331</width>
     <height>501</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Double click on a part to add it to the circuit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Generic Parts</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>331</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Templates for ICs and User-Defined Parts</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>580</y>
     <width>321</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Updates the templates with parts from an existing circuit</string>
   </property>
   <property name="text">
    <string>Add from VRT file</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
